kpatch-build/create-diff-object.c

```diff
@@ -2467,8 +2467,17 @@ static int function_ptr_rela(const struct rela *rela)
 
 static int may_need_dynrela(const struct rela *rela)
 {
+	/*
+	 * References to .TOC. are treated specially by the module loader and
+	 * should never be converted to dynrelas.
+	 */
+	if (rela->type == PPC64_REL16_HA || rela->type PPC64_REL16_LO ||
```

Comment:
> Yes, they are not used for other relocations. The v2 looks good, '==' missing in above check.

Reply:
Doh, guess I should have at least compiled it.